debian/control

```diff
@@ -0,0 +1,39 @@
+Source: libcontentfeed
+Priority: standard
+Maintainer: Sam Spilsbury <sam@endlessm.com>
+Build-Depends: debhelper (>= 9),
+               dh-autoreconf,
+               gobject-introspection,
+               libeos-shard-0-dev,
+               libgirepository1.0-dev,
```

Comment:
I don't think this is a dependency required to build libcontentfeed... Typically that library is used by bindings like gjs or pygobject.

Reply:
Yeah, fair enough. For some reason it was in the e-d-f package, maybe its not needed there either...